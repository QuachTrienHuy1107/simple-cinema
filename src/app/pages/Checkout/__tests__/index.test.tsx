import * as React from "react";
import { render } from "@testing-library/react";

import { Checkout } from "..";

jest.mock("react-i18next", () => ({
    useTranslation: () => {
        return {
            t: str => str,
            i18n: {
                changeLanguage: () => new Promise(() => {}),
            },
        };
    },
}));

describe("<Checkout  />", () => {
    it("should match snapshot", () => {
        const loadingIndicator = render(<Checkout />);
        expect(loadingIndicator.container.firstChild).toMatchSnapshot();
    });
});
