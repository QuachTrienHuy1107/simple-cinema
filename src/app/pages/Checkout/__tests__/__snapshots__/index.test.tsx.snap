// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Checkout  /> should match snapshot 1`] = `
<div
  class=""
/>
`;
